src/Features/Core/Portable/Completion/CompletionServiceWithProviders.cs

```diff
@@ -11,7 +11,9 @@
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Completion.Providers;
+using Microsoft.CodeAnalysis.ExtractMethod;
```

Comment:
let me guess, import completion did this?

Reply:
yep!